pkg/version/BUILD

```diff
@@ -32,3 +31,36 @@ go_test(
     tags = ["automanaged"],
     deps = [],
 )
+
+# this file replace base.go in this package and is integrated into the
+# bazel build.
+genrule(
+    name = "base_generated",
+    outs = [
+        "base_generated.go",
+    ],
+    cmd = """
+set -o errexit
+set -o nounset
+set -o pipefail
+
+eval $$(cat bazel-out/volatile-status.txt | tr ' ' '=')
```

Comment:
maybe `$bazel_out` instead of `bazel-out`? it appears in `bazel info --show_make_env`, but I'm not sure if it's a real var.